frontend/packages/core/src/AppLayout/header.tsx

```diff
@@ -40,7 +48,12 @@ const StyledLogo = styled("img")({
   verticalAlign: "middle",
 });
 
-const Header: React.FC<AppConfiguration> = ({ title = "clutch", logo = <Logo /> }) => {
+const Header: React.FC<HeaderProps> = ({
+  title = "clutch",
+  logo = <Logo />,
+  showNPS = false,
+  showShortLinks = false,
```

Comment:
Aren't shortlinks only visible if the page supports it? Would it be weird to have this conditional be true but have links not show on some pages? Maybe we should call this something like `enableShortLinks` instead of show just to make it clear we are enabling functionality but it doesn't mean that there will always be short links shown.

Reply:
Actually good call out, it wont work anyways since the ShortLinks component actually does the check and returns null if it's not a valid workflow. I'll remove it.